packages/opentelemetry-instrumentation-langchain/tests/test_documents_chains.py

```diff
@@ -36,17 +36,16 @@ def test_sequential_chain(exporter):
     small_docs = CharacterTextSplitter().create_documents(texts=[INPUT_TEXT, ])
     llm = ChatCohere(model="command", temperature=0.75)
     chain = load_summarize_chain(llm, chain_type="stuff")
-    chain.run(small_docs)
+    chain.invoke(small_docs)
 
     spans = exporter.get_finished_spans()
 
     assert [
-        "ChatCohere.langchain.task",
-        "LLMChain.langchain.task",
-        "StuffDocumentsChain.langchain.task",
+        "LLMChain.langchain.workflow",
```

Comment:
Ideally, this should be a task inside the `StuffDocumentsChain` workflow

Reply:
Both `LLMChain` and `StuffDocumentsChain` inherit from `Chain` (also `SequentialChain` - see test_sequential_chain), which `kind` is defined as `WORKFLOW` in `WRAPPED_METHODS`.

One possibility would be to get rid of the `"kind"` here and make some logic inside callback_wrapper, e.g. if there is a `parent_run_id`, it's a `TASK`.

Another possibility would be to list the classes and assign them the kind explicitly.

I went with the current solution because it was simple and the least coupling to the langchain implementation. Let me know what's your preference please!

Reply:
I think the solution you suggested, where you check for `parent_run_id` sounds good and better. The expectation is for the `workflow` to only be that root span.

Reply:
@nirga Thanks for feedback, added this logic in commit 27024dd3b6a3ea5d26aafd7dfea2c9971eb9ba1b.

Additionally, I added a second commit (5fa768928f72e7c109fba4ecf0ea6deaf66b0c96) which removes "kind" from __init__ and passing it to the constructor, because imho it is a good simplification now. Let me know if you agree with this one, otherwise I can revert.